messaging/cloud-events/amqp-json/src/main/java/io/quarkus/ts/messaging/amqpreactive/Resource.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.ts.messaging.amqpreactive;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/")
+public class Resource {
+
+    @Inject
+    Consumer c;
+
+    @GET
+    @Path("/result")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response result() {
+        return Response.ok().entity(c.getResults()).build();
```

Comment:
also if you want a less verbose way to return an entity you could use:
`Response.ok(c.getResults()).build()`

Or instead of return a response just return a `Map<String,String>`

```
 @GET
    @Path("/result")
    @Produces(MediaType.APPLICATION_JSON)
    public Map<String,String> result() {
        return c.getResults();
    }
```